docs_app/src/styles/1-layouts/_marketing-layout.scss

```diff
@@ -58,6 +58,10 @@ section#intro {
     justify-content: center;
     align-items: center;
 
+    .hero-cta + .hero-cta {
```

Comment:
Does it not work just to use `.hero-cta` without the sibling?

Reply:
It would work both ways. I was specifically trying to target the second `.hero-cta` to give it some space from the first. Removing the sibling selector would cause the "Reactive Extensions Library for JavaScript" to get pushed upwards.

Sibling Selector:
![image](https://user-images.githubusercontent.com/17665837/47737389-34b70b00-dc47-11e8-86f8-7b4705842fe3.png)

No sibling selector:
![image](https://user-images.githubusercontent.com/17665837/47737508-7778e300-dc47-11e8-867e-208b105dbb9d.png)

There isn't a big difference. If it's preferrable to apply the styling to both buttons I can certainly remove the sibling selector.

Reply:
Nah it's fine like it is, thanks a lot!